server/tests/integration/customers.test.ts

```diff
@@ -89,9 +99,33 @@ describe('Customers endpoints', () => {
       expect(res.body.error.message).toBe('Missing Authorization token.');
     });
 
+    test('Should call customer access control', async () => {
+      mockCustomerAuth();
+
+      await request(app).post('/customers');
+
+      expect(customerAccessControl).toHaveBeenCalledTimes(1);
+    });
+
+    test('Should reject if customer is not authorized', async () => {
+      mockCustomerAuth();
+
+      const expectedCustomerData = customerFixtures.responseData?.[0];
+      const gpayId = expectedCustomerData.gpayId;
+
+      const res = await request(app).post('/customers').send({gpayId});
+
+      expect(res.status).toBe(403);
+      expect(res.body).toHaveProperty('error');
+      expect(res.body.error.message).toBe(
+        'Not allowed to modify or access requested resource.'
```

Comment:
where did you set gpayid from spot identity api/token?

Reply:
@ifhuang Oh you mean for the test? It is in the customer-auth mock which is in PR #212. Basically I just added a dummy sub for the mock.
![image](https://user-images.githubusercontent.com/25261058/90473771-489e6a80-e156-11ea-8a2a-2b814f7fff96.png)